src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -38,6 +37,13 @@
     <ProjectReference Include="..\..\harness\BenchmarkDotNet.Extensions\BenchmarkDotNet.Extensions.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Why do we need this here?

Reply:
I needed this so that locally MicroBenchmarks project could pick up my new API (FirstSpan)

According to [benchmarking-workflow-corefx.md#PR](https://github.com/dotnet/performance/blob/master/docs/benchmarking-workflow-corefx.md#PR),
and from my understanding, after my corefx PR (with the new API) gets merged and a new NuGet package is published to the CoreFX NuGet feed, I can remove the reference below to System.Memory.dll and install/update the package consumed by MicroBenchmarks

Reply:
After this is merged, could you please open an issue to track this?

Reply:
Does it need to be guarded under kind of "netcoreapp3.0"?

Reply:
> After this is merged, could you please open an issue to track this?

I don't think I would be merging this PR which is on performance repo, until the corefx repo containing the new API gets merged.
Once that happens I think I should add another commit here that install the correct corefx nuget package for System.Memory.

> Does it need to be guarded under kind of "netcoreapp3.0"?
cc: @adamsitnik what is your recommendation?

Reply:
> I don't think I would be merging this PR which is on performance repo

Right. I don't think the intention was to merge this change. For instance the hint path is specific to @maryamariyan's local build/directories.

> Does it need to be guarded under kind of "netcoreapp3.0"?

I believe so. See my previous comment: https://github.com/dotnet/performance/pull/410/files#r270154876